terratag.go

```diff
@@ -23,10 +24,11 @@ import (
 )
 
 type counters struct {
-	totalResources  uint32
-	taggedResources uint32
-	totalFiles      uint32
-	taggedFiles     uint32
+	totalResources    uint32
+	taggedResources   uint32
+	totalFiles        uint32
+	taggedFiles       uint32
+	untaggedResources uint32 // For dryrun.
```

Comment:
counts the number of untagged resources (if after dryrun there are still untagged resources - exit 1 is returned).